lib/MusicBrainz/Server/Data/WikiDoc.pm

```diff
@@ -38,7 +38,7 @@ sub _fix_html_links
         {
             # Transform link on "zoomable" image to point directly to the original image
             my $href = $child->attr('src') || '';
-            $href =~ s,^$WIKI_IMAGE_PREFIX/thumb,//$wiki_server$WIKI_IMAGE_PREFIX,;
+            $href =~ s,^$WIKI_IMAGE_PREFIX(?:/thumb)?,//$wiki_server$WIKI_IMAGE_PREFIX,;
             $href =~ s,/[0-9]+px-[^/]*$,,;
```

Comment:
The next line also assumed that it was a thumbnail URL, it seems safer to combine these together.

```suggestion
            $href =~ s,^$WIKI_IMAGE_PREFIX,//$wiki_server$WIKI_IMAGE_PREFIX,;
            $href =~ s,^(//$wiki_server$WIKI_IMAGE_PREFIX)/thumb(.*)/[0-9]+px-[^/]*$,$1$2,;
```

Reply:
Sounds like a good safety measure, tested and it works (even though it confuses the poor IDE which expects apparently unneeded escapes).